frontend/src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -34,7 +46,7 @@ const ModalWindow: FC<IChange> = ({ type, message, date, expDate, idd }) => {
       /[0-9]+/.test(input.expDate)
     ) {
       if (!type) {
-        dispatch(addModalTodo(input));
+        addMongoTodo(input.message, input.date, input.expDate);
```

Comment:
Why do you remove 'input' as param ? You can use destructuring in addMongoTodo function